src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
```

Comment:
> Define a slice of state for books that:
Includes an array of books (initial state: empty array)

- [OPTIONAL] You have done great making use of redux slice. You had this in your previous code but it seems you removed it as you updated your code. looking at your initial state, according to the requirement, your initial state needs to be an object with a `books` state initialized to an empty array, I'll suggest you actually update this before merging this pull request. :+1: :100: 
```javascript
  initialState: {
    books: [],
  },
```